change/@microsoft-teams-js-277568ef-041e-4cc5-8169-6c0f16f39cef.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Embedded apps no longer incorrectly get their parameters wrapped in an array",
+  "packageName": "@microsoft/teams-js",
+  "email": "sigulati@microsoft.com",
+  "dependentChangeType": "minor"
```

Comment:
@TrevorJoelHarris , I changed the "dependentChangeType" to minor. I am guessing that is ok?